app/src/main/java/org/p2p/wallet/common/feature_toggles/di/FeatureTogglesModule.kt

```diff
@@ -14,7 +14,7 @@ import org.p2p.wallet.common.feature_toggles.toggles.remote.SslPinningFeatureTog
 
 object FeatureTogglesModule : InjectionModule {
     override fun create() = module {
-        singleOf(::AppFirebaseRemoteConfig)
+        single(createdAtStart = true) { AppFirebaseRemoteConfig() }
```

Comment:
now RemoteConfig is initialised when the app starts without any explicit method call in `App` or other place